modules/nf-core/getorganelle/config/main.nf

```diff
@@ -0,0 +1,44 @@
+process GETORGANELLE_CONFIG {
+    tag "${organelle_type}"
+    label 'process_single'
+
+    conda "bioconda::getorganelle=1.7.7.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/getorganelle:1.7.7.0--pyh7cba7a3_0':
+        'biocontainers/getorganelle:1.7.7.0--pyh7cba7a3_0' }"
+
+    input:
+    val(organelle_type)
+
+    output:
+    tuple val(organelle_type), path("getorganelle"), emit: db
+    path "versions.yml"                            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    get_organelle_config.py \\
+        -a ${organelle_type} \\
+        --config-dir getorganelle
```

Comment:
```suggestion
    get_organelle_config.py \\
        -a ${organelle_type} \\
        --config-dir getorganelle $args
```

Reply:
Sorry! I forgot to add it! I've decided to put it on a different line, though.